tests/cases/unittests/session.ts

```diff
@@ -0,0 +1,459 @@
+/// <reference path="..\..\..\src\harness\harness.ts" />
+
+module ts.server {
+	let lastWrittenToHost: string,
+	mockHost: ServerHost = {
+		args: [],
+		newLine: '\n',
+		useCaseSensitiveFileNames: true,
+		write: (s) => lastWrittenToHost = s,

```

Comment:
The TS emit doesn't change to stop that when the function is typed to return void? Huh. I kinda assumed that was a thing. Will do.